README.rst

```diff
@@ -62,14 +62,16 @@ Compatibility
 
 llvmlite works with Python 3.6 and greater.
 
-As of version 0.33.0, llvmlite requires LLVM 9.0.x.
+As of version 0.34.0, llvmlite requires LLVM 10.0.x. on all architectures
+except ``aarch64`` which requires LLVM 9.0.x.
```

Comment:
Should probably say why?

Reply:
Ideally, I would like to reference an LLVM bug report here.

Reply:
https://bugs.llvm.org/show_bug.cgi?id=46996